ITE/1/README.adoc

```diff
@@ -0,0 +1,925 @@
+  ITEs which will not include code changes may omit this section.
+. **References** -- When moving a ITE from a Draft to "Accepted" or "Final" state,
+  the references section must be updated to include links to the pull requests
+  and mailing list discussions which were involved in the process. The ITE
+  should self-document the process in which it was developed.
+
+WARNING: ITE submissions that do not adequately complete any of the above
+sections will not be approved as ITE Drafts.
+
+The final implementation must include test code and documentation
+appropriate for either the Jenkins user or developer documentation.
+
+==== ITE File Format
+
+ITEs are UTF-8 encoded text files using the
+link:https://asciidoctor.org[AsciiDoc] format.
+AsciiDoc allows for rich markup that is still quite easy to read,
+but can also be used to render good-looking and functional HTML.
+
+[[metadata]]
+==== ITE Header Preamble
+
+===== Required Metadata
+
+All ITEs MUST begin with an AsciiDoc table containing metadata relevant to the
+ITE:
+
+[source,asciidoc]
+----
+.Metadata
+[cols="2"]
+|===
+| ITE
+| 1
+
+| Title
+| in-toto Enhancement Format
+
+| Sponsor
+| link:https://github.com/santiagotorres[Santiago Torres-Arias]
+
+| Status
+| Draft :speech_balloon:
+
+| Type
+| Process
+
+| Created
+| 2018-10-04
+|===
+----
+
+
+. **ITE** -- ITE number, given by the ITE editors. Use `0000` until one is assigned.
+. **Title** -- Brief title explaining the proposal in fewer than 50 characters
+. **Sponsor** -- <<Sponsor>> of the ITE, in essence, the individual
+  responsible for seeing the ITE through the process.
+. **Status** -- Draft :speech_balloon:, Deferred :hourglass:, Accepted :ok_hand:,
+  Rejected :no_entry:, Withdrawn :hand:, Final :lock:, Replaced :dagger:, Active :smile:.
+. **Type** -- Describes the type of ITE: Standards, Informational, Process
+. **Created** -- Date (`%Y-%m-%d`) when the document was first created.
+
+=====  Additional Header Rows
+
+Issue:: [[header-issue]]
+An **issue identifier** row is available to specify a linked placeholder for an
+issue in github, if any.
+
+Discussions-To:: [[header-discussions-to]]
+For a ITE where final pronouncement will be made on a list other than
+in-toto-dev@googlegroups.com or the github issue tracker, a **Discussions-To**
+row will indicate the mailing list or URL where the pronouncement will occur. A
+temporary Discussions-To header may also be used when a draft ITE is being
+discussed prior to submission for pronouncement.
+
+Requires:: [[header-requires]]
+ITEs may have a **Requires** row, indicating the ITE numbers that this
+ITE depends on.
+
+Superseded-By:: [[header-superseded-by]]
+ITEs may also have a **Superseded-By** row indicating that a ITE has been
+rendered obsolete by a later document; the value is the number of the ITE that
+replaces the current document. The newer ITE must have a **Replaces** row
+containing the number of the ITE that it rendered obsolete.
```

Comment:
Well, only if it replaces anything no? Or would you rather have "none" or so in that row if it replaces anything?

Reply:
I meant add in order to describe, like you are describing all the other available "Additional header rows" here, e.g something like
```
Replaces:: [[header-replaces]]
An ITE that replaces another ITE must have a **Replaces** row containing the number of the ITE that it rendered obsolete.
```

(Not add because this particular ITE replaces another ITE) :)

Reply:
ok, will add